### ISSUE TITLE
What does err_L1 do?

### ISSUE THREAD

**Issue by Jingyilang** [PKU-ICST-MIPL/CM-GANS_TOMM2019#3]
It seems like err_L1 is just to print? But in annotation, It says "L1 loss which forces features generated by TG and IG to be closer". How does it do that?

**Comment by PKU-ICST-MIPL** [comment]
Indeed, L1 loss may force generated features to be closer. But we didn't add it into final loss function when conduct backward propagation, you can simply ignore it.
